test/parallel/test-npm-install.js

```diff
@@ -57,7 +57,5 @@ function handleExit(error, stdout, stderr) {
 
   assert.strictEqual(code, 0, `npm install got error code ${code}`);
   assert.strictEqual(signalCode, null, `unexpected signal: ${signalCode}`);
-  assert.doesNotThrow(function() {
-    fs.accessSync(`${installDir}/node_modules/package-name`);
-  });
+  assert(common.fileExists(`${installDir}/node_modules/package-name`));
```

Comment:
`fs.existsSync()` would be a better choice.

Reply:
This raises the question of whether `common.fileExists` is very aptly named or not... it uses `accessSync` under the hood rather than `existsSync`.

Reply:
I mentioned this somewhere else recently. `fs.existsSync()` was marked for deprecation a while back, so `common.fileExists()` was added to take its place in the tests. After enough user ~~outrage~~ feedback, `existsSync()` was undeprecated. `common.fileExists()` is now redundant and should be removed #codeandlearn #goodfirstcontribution.

Reply:
@cjihrig @apapirovski Thank you for review.
Should I replace `common.fileExists` with `fs.existsSync` ?

Reply:
@Leko I would change it.

Reply:
I got it. I'll update it.